common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -17,13 +17,15 @@
 class EntitlementViewSet(viewsets.ModelViewSet):
     authentication_classes = (JwtAuthentication, SessionAuthentication,)
     permission_classes = (permissions.IsAuthenticated, permissions.IsAdminUser,)
-    queryset = CourseEntitlement.objects.all().select_related('user')
     lookup_value_regex = '[0-9a-f-]+'
     lookup_field = 'uuid'
     serializer_class = CourseEntitlementSerializer
     filter_backends = (DjangoFilterBackend,)
     filter_class = CourseEntitlementFilter
 
+    def get_queryset(self):
+        return CourseEntitlement.objects.all().select_related('user')
```

Comment:
What does this do?

Reply:
Why this VS the original?

Reply:
AJ needed this as well, it had to do with this `get_queryset` being called on every request rather than just on server start with `queryset`. It was needed since each request would then use the appropriate expiry policy, but that route will no longer work. With those changes to not using the Manager any more I won't need this.